tests/components/samsungtv/test_media_player.py

```diff
@@ -1393,4 +1378,25 @@ async def test_upnp_missing_service(
         {ATTR_ENTITY_ID: ENTITY_ID, ATTR_MEDIA_VOLUME_LEVEL: 0.6},
         True,
     )
-    assert f"Upnp service {UPNP_SVC_RENDERING_CONTROL} is not available" in caplog.text
+    assert "Upnp services are not available" in caplog.text
+
+
+@pytest.mark.usefixtures("remotews")
```

Comment:
As per https://github.com/home-assistant/core/pull/68172, `rest_api` should probably be set explicitely
```suggestion
@pytest.mark.usefixtures("remotews", "rest_api")
```